packages/auth/amplify_auth_cognito/lib/src/CognitoSignIn/CognitoSignInOptions.dart

```diff
@@ -17,10 +17,13 @@ import 'package:amplify_core/amplify_core.dart';
 
 class CognitoSignInOptions extends SignInOptions {
   Map<String, String>? clientMetadata;
-  CognitoSignInOptions({this.clientMetadata}) : super();
+  final AuthenticationFlowType? authFlowType;
```

Comment:
How about just `authFlow`. The `type` suffix originates in the SDK where they add `Type` to the end of all types for some reason